velox/functions/lib/MapFromEntries.cpp

```diff
@@ -139,12 +142,14 @@ class MapFromEntriesFunction : public exec::VectorFunction {
           // Check nulls in the top level row vector.
           const bool isMapEntryNull = decodedRowVector->isNullAt(offset + i);
           if (isMapEntryNull) {
-            // Set the sizes to 0 so that the final map vector generated is
-            // valid in case we are inside a try. The map vector needs to be
-            // valid because its consumed by checkDuplicateKeys before try
-            // sets invalid rows to null.
+            // The map vector needs to be valid because its consumed by
+            // checkDuplicateKeys before try sets invalid rows to null.
```

Comment:
The comment looks a bit confusing to me because it seems to not match the implementation. Could you please confirm?